packages/paste-website/src/pages/getting-started/design.mdx

```diff
@@ -48,7 +48,7 @@ However, this release is not:
 
 ### Basic Terminology
 
-If you’re unfamiliar with Abstract, version control, or git workflows, read through <a href="https://www.goabstract.com/how-it-works/">how it works</a>.
+If you’re unfamiliar with Abstract, version control, or git workflows, read through <a href="https://www.goabstract.com/how-it-works/">how it works</a>. Abstract assumes you have working knowledge of version control since they use version control terms. Brushing up on these would be a good first step.
```

Comment:
"Abstract assumes..." this line can turn people off reading this section. Would it be better instead to say "Brush up on these terms first!"